tests/sentry/integrations/vercel/test_webhook.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+
+from sentry.testutils import APITestCase
+from sentry.testutils.helpers import override_options
+
+from .testutils import EXAMPLE_DEPLOYMENT_WEBHOOK_RESPONSE
+
+signature = "74b587857986545361e8a4253b74cd6224d34869"
+secret = "AiK52QASLJXmCXX3X9gO2Zyh"
+
+
+class VercelWebhookTest(APITestCase):
+    def setUp(self):
+        self.url = "/extensions/vercel/webhook/"
+
+    def test_get(self):
+        response = self.client.get(self.url)
+        assert response.status_code == 405
```

Comment:
@MeredithAnya Where does the `405` status code come from?

Reply:
@scefali I'm pretty sure it comes from the django `APIView` when you don't have the route defined. In other places we return `HttpResponse(status=405)` but  that's with using the generic `View`

Reply:
@MeredithAnya I see `405` means `Method not allowed`, so this test is basically saying you can't do a `GET` request on this route. Not sure why we need this test.

Reply:
I was just seeing that we test this for other webhook endpoints 🤷